wandb/sdk/launch/runner/sagemaker_runner.py

```diff
@@ -204,6 +208,10 @@ def run(
 
         launch_project.fill_macros(image_uri)
         _logger.info("Connecting to sagemaker client")
+        entry_point = (
+            launch_project.override_entrypoint
+            or launch_project.get_single_entry_point()
+        )
```

Comment:
sort of weird that the container_runner uses `launch_project.override_entrypoint.command` while this is `launch_project.override_entrypoint`, i guess we would have to change `build_sagemaker_args` which could be annoying idk...

Reply:
actually it would only require changing `merge_image_uri_with_algorithm_specification` to accept a str, and: 

```
if entrypoint:
        algorithm_specification["ContainerEntrypoint"] = entrypoint.command
```

would become:

```
if entrypoint_command:
        algorithm_specification["ContainerEntrypoint"] = entrypoint_command
```